packages/neos-ui/src/Containers/Modals/SelectNodeType/index.js

```diff
@@ -189,10 +192,17 @@ export default class SelectNodeType extends PureComponent {
     }
 
     renderSelectNodeTypeDialogHeader() {
+        const {insertMode} = this.state;
+        const {i18nRegistry, nodeTypesRegistry, referenceNodeType} = this.props;
+        const nodeTypeLabel = $get('ui.label', nodeTypesRegistry.get(referenceNodeType))
+        const nodeTypeLabelText = i18nRegistry.translate(nodeTypeLabel, 'Node')
+        const addLabel = i18nRegistry.translate('Neos.Neos.Ui:Main:add', 'Add')
+
+        const insertModeLabel = i18nRegistry.translate('Neos.Neos.Ui:Main:InsertModeTitle' + insertMode, 'to')
```

Comment:
i would strongly recommend to not use string concatenation for labels, as its impossible to find them in the source code ^^ so im proposing a insertMode to label map or switch case (doesnt matter) but future us will thank us :D

Reply:
I updated the PR with a switch case 👍